src/main/java/data/wishlist.json

```diff
@@ -0,0 +1,32 @@
+{"games": [
+  {
+    "gameID": "RAILROADS Online!",
+    "metacriticScore": "Very Positive",
+    "salePrice": "34.99",
+    "thumb": "235278",
+    "steamRatingCount": "10.0",
+    "title": "24.91",
+    "steamRatingPercent": "4693",
+    "dealRating": "https://shared.fastly.steamstatic.com/store_item_assets/steam/apps/1696810/capsule_sm_120.jpg?t=1731666585",
+    "normalPrice": "1",
+    "storeName": "https://www.gamersgate.com",
+    "savings": "unavailable",
+    "isOnSale": "28.808231",
+    "steamRatingText": "81"
+  },
+  {
+    "gameID": "Back 4 Blood Ultimate",
+    "metacriticScore": "unavailable",
+    "salePrice": "99.99",
+    "thumb": "293097",
+    "steamRatingCount": "10.0",
+    "title": "8.89",
+    "steamRatingPercent": "unavailable",
+    "dealRating": "https://www.indiegalacdn.com/get_store_img?img=store-img_game/games/medium/924970_ultimate.jpg&s=medium",
+    "normalPrice": "1",
+    "storeName": "https://www.indiegala.com",
+    "savings": "unavailable",
+    "isOnSale": "91.109111",
+    "steamRatingText": "unavailable"
+  }
+]}
```

Comment:
Can you not commit changes to the wishlist on a PR to the main branch

Reply:
done, thank you for the reminder!